src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs

```diff
@@ -181,52 +186,49 @@ public override ExportResult Export(in Batch<Metric> batch)
                                     MetricEventType.DoubleMetric,
                                     metric.Name,
                                     metricPoint.EndTime.ToFileTime(),
                                     metricPoint.Tags,
-                                    metricData);
-                                this.metricDataTransport.Send(MetricEventType.DoubleMetric, this.bufferForNonHistogramMetrics, bodyLength);
+                                    metricData,
+                                    exemplars);
+                                this.metricDataTransport.Send(MetricEventType.TLV, this.buffer, bodyLength);
                                 break;
                             }
 
                         case MetricType.DoubleGauge:
                             {
                                 var doubleSum = metricPoint.GetGaugeLastValueDouble();
                                 var metricData = new MetricData { DoubleValue = doubleSum };
-                                var bodyLength = this.SerializeMetric(
+                                var bodyLength = this.SerializeMetricWithTLV(
                                     MetricEventType.DoubleMetric,
                                     metric.Name,
                                     metricPoint.EndTime.ToFileTime(),
                                     metricPoint.Tags,
-                                    metricData);
-                                this.metricDataTransport.Send(MetricEventType.DoubleMetric, this.bufferForNonHistogramMetrics, bodyLength);
+                                    metricData,
+                                    exemplars);
+                                this.metricDataTransport.Send(MetricEventType.TLV, this.buffer, bodyLength);
                                 break;
                             }
 
                         case MetricType.Histogram:
                             {
-                                var sum = new MetricData { UInt64Value = Convert.ToUInt64(metricPoint.GetHistogramSum()) };
+                                var sum = Convert.ToUInt64(metricPoint.GetHistogramSum());
                                 var count = Convert.ToUInt32(metricPoint.GetHistogramCount());
-                                MetricData min = ulongZero;
-                                MetricData max = ulongZero;
-                                if (metricPoint.TryGetHistogramMinMaxValues(out var minValue, out var maxValue))
-                                {
-                                    min = new MetricData { UInt64Value = Convert.ToUInt64(minValue) };
-                                    max = new MetricData { UInt64Value = Convert.ToUInt64(maxValue) };
-                                }
-
-                                var bodyLength = this.SerializeHistogramMetric(
+                                _ = metricPoint.TryGetHistogramMinMaxValues(out double min, out double max);
```

Comment:
if min max is not available, dont we need ulongZero to be written?

Reply:
Yes, it would serialize them as zero. If `TryGetHistogramMinMaxValues` returns false, min and max would be set to zero.

Reply:
mm.. Is that a contract or relying on the internal implementation details on OTel sdk? I think its wrong to rely on the min, max if TryGet fails.

Reply:
That should be considered a breaking change from the SDK standpoint: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Metrics/MetricPoint.cs#L328-L329

Reply:
I am not sure if that is the case. Once it returned false, callers should not be relying on the values on the out parameters.

Reply:
I can explicitly set them to zero if the method returns false, but I do think the SDK should not change the behavior of this public API when it returns false. Also, it doesn't make sense for the SDK to assign to them any value other than the default value of zero when it returns false.